webpack.config.js

```diff
@@ -53,11 +53,6 @@ Encore.setOutputPath('./src/Resources/public')
 
   .copyFiles([
     { from: './assets/images/', pattern: /\.(png|gif)$/, to: 'images/[name].[ext]' },
-    {
```

Comment:
I guess it's important to skins? 

because I'm getting this error

```
GET http://localhost:8001/bundles/sonataadmin/admin-lte-skins/skin-black.min.css net::ERR_ABORTED 404 (Not Found)
```

Reply:
I didn't find the skins on the admin-lte v3, maybe we need to remove that feature too.

Reply:
I founded this

https://adminlte.io/docs/3.1//additional-styles.html